test/intl402/DateTimeFormat/prototype/formatRangeToParts/en-US.js

```diff
@@ -27,10 +27,10 @@ function compare(actual, expected) {
   }
 }
 
-const date1 = new Date("2019-01-03");
-const date2 = new Date("2019-01-05");
-const date3 = new Date("2019-03-04");
-const date4 = new Date("2020-03-04");
+const date1 = new Date("2019-01-03T00:00:00Z");
```

Comment:
If you use "2019-01-03T00:00:00Z" here, it mean the date value is in UTC. But when you format the date, the date will be output according to local timezone, therefore it won't be 2019-01-03 in some timezone the machine is running. I think you need to use "2019-01-03T00:00:00" or "2019-01-03T00:00" instead so the date value will be in locale timezone of 2019-01-03. For example, my machine is in PST. if you use  "2019-01-03T00:00:00Z" then date 1 is UTC of 2019-01-03 00:00 which is 2019-01-02 16:00:00 in PST. then when you format it, sine the dft is not created for a particular timezone, it will use local time zone (PST) to format, and therefore output as 2019/1/2 instead of 2019/1/3 as what you expected. In the other hand, if you use new Date("2019-01-03T00:00") , ECMA 262 said it will be interpret as using local time zone. Notice new Date("2019-01-03"), which you previously use, will be created as in UTC, not local timezone.

Reply:
Ugh.